src/frontend/sharedComponents/ProjectInviteBottomSheet/index.tsx

```diff
@@ -64,9 +65,23 @@ export const ProjectInviteBottomSheet = ({
   const accept = useAcceptInvite();
   const reject = useRejectInvite();
 
-  if (invite && !inviteIsOpen && enabledForCurrentScreen) {
-    openInviteSheet();
-  }
+  React.useEffect(() => {
+    if (
+      (invite || acceptedInvite) &&
+      !inviteModalVisible &&
+      enabledForCurrentScreen
+    ) {
+      setInviteModalVisible(true);
+    }
+  }, [invite, acceptedInvite, inviteModalVisible, enabledForCurrentScreen]);
```

Comment:
i think this can be taken our of a useEffect and you can just do the If statement directly in line

Reply:
@ErikSin I tried removing it from the useEffect but it made it so that the invite success modal would not close.